src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    public static final int COUNT_BALL = 3;
+    public static final int MAX_POSSIBLE_NUMBER_FOR_BALL = 100;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball [] balls = new Ball[COUNT_BALL];
+        for (Ball resultBall: balls) {
+            System.out.println(lottery.getRandomBall());
+        }
```

Comment:
Array isnt used here properly.
```suggestion
        for (int i = 0; i < COUNT_BALL; i++) {
            System.out.println(lottery.getRandomBall());
        }
```